spec/index.bs

```diff
@@ -5170,70 +5187,112 @@ enum GPUStoreOp {
 <dl dfn-type=method dfn-for=GPURenderEncoderBase>
     : <dfn>setPipeline(pipeline)</dfn>
     ::
+        Sets the current {{GPURenderPipeline}}.
 
         <div algorithm="GPURenderEncoderBase.setPipeline">
             **Called on:** {{GPURenderEncoderBase}} this.
 
             **Arguments:**
             <pre class=argumentdef for="GPURenderEncoderBase/setPipeline(pipeline)">
-                pipeline:
+                |pipeline|: The render pipeline to use for subsequent drawing commands.
             </pre>
 
             **Returns:** void
 
-            Issue: Describe {{GPURenderEncoderBase/setPipeline()}} algorithm steps.
+            Issue the following steps on the [=Device timeline=] of |this|:
+            <div class=device-timeline>
+                1. If any of the following conditions are unsatisfied, make |this| [=invalid=] and stop.
```

Comment:
There should also be validation that the pipeline is compatible with the descriptor of the render pass (e.g. format and sample count of attachment matches). Since this requires the `GPURenderPassEncoder` to remember its descriptor, we could have an `Issue:` and deal with it later.

Reply:
Added the issue.